variables.tf

```diff
@@ -118,20 +118,8 @@ variable "kms_alias_name" {
   type        = string
 }
 
-variable "ami_name_filter" {
-  description = "Name filter to help pick the AMI."
-  default     = ["vault-consul-ubuntu18-*"]
-  type        = list(string)
-}
-
 variable "ami_id" {
   description = "ID of the AMI to be used for the Consul and Vault instances."
-  type        = string
-  default     = ""
-}
-
-variable "ami_owner" {
-  description = "AWS account ID of the AMI owner. Defaults to HashiCorp."
-  default     = "562637147889"
+  default     = null
```

Comment:
I have two notions:
1. Are Vault and Consul AMIs all the time have the same ID?
2. This parameter should not have default value because it is required.

Reply:
1. Will be handling the ID from here: https://github.com/Flaconi/ops-infra-core/pull/1972/files#diff-c7b9b7814f6bf86acd86775ee1edcc9169de682b2455e1a40535e740137e1228R83
2. Fixed